parcel-resolver-veda/stringify-yml-func.js

```diff
@@ -38,7 +46,15 @@ const md = markdownit();
  * @returns string
  */
 function stringifyYmlWithFns(data, filePath) {
-  const jsonVal = JSON.stringify(data, (k, v) => {
+  const mdxData = {
+    ...data,
+    layers: data.layers?.map((l) => ({
+      ...l,
+      parentDataset: getParentDataset(data)
```

Comment:
```suggestion
      parentDataset: data.id
```

Reply:
We would need to find the places where this is being used in the UI!

Reply:
Yah this would require more changes but more I think this is the right approach.